src/nomad_simulations/schema_packages/properties/thermodynamics.py

```diff
@@ -235,3 +236,31 @@ class Hessian(PhysicalProperty):
         description="""
         """,
     )
+
+    n_negative_eigenvalues = Quantity(
+        type=positive_int(),
+        description="""
+        Number of negative Hessian eigenvalues (imaginary vibrational frequencies).
+        A value of 0 indicates a local minimum, 1 a first-order saddle point, and
+        >1 a higher-order saddle point. Leave unset if the Hessian was evaluated
+        away from a stationary point.
+        """,
+    )
```

Comment:
Could we sort this quantity in descending order during normalization?
When running phonon calculations (via displacement) in solid state, it is common for very low-frequency artifacts to show up. This is due to the complexity of some grids to fully relax.
Under 100 cm-1 these are considered acceptable numerical errors. The important thing is to see the main frequency.